packages/components/forms/src/input-group/InputGroup.test.tsx

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe } from '@/scripts/test/axeHelper';
+
+import { InputGroup } from './InputGroup';
+import { TextInput } from '@contentful/f36-forms';
+import { IconButton } from '@contentful/f36-button';
+import { LockIcon } from '@contentful/f36-icons';
+
+describe('InputGroup', function () {
+  it('renders the component', () => {
+    const { getByRole, getByLabelText } = render(
+      <InputGroup>
```

Comment:
Shouldn't it be `TextInputGroup` since it's applicable only for `TextInput` component?

Reply:
Or even maybe `TextInput.Group`?